USAGE.md

```diff
@@ -81,12 +77,10 @@ code will look something like this:
 2. Initialize the Python interpreter:
    ```swift
    guard let pythonHome = Bundle.main.path(forResource: "python", ofType: nil) else { return }
-   guard let pythonPath = Bundle.main.path(forResource: "python/lib/python3.13", ofType: nil) else { return }
-   guard let libDynloadPath = Bundle.main.path(forResource: "python/lib/python3.13/lib-dynload", ofType: nil) else { return }
    let appPath = Bundle.main.path(forResource: "app", ofType: nil)
 
    setenv("PYTHONHOME", pythonHome, 1)
-   setenv("PYTHONPATH", [pythonPath, libDynloadPath, appPath].compactMap { $0 }.joined(separator: ":"), 1)
+   setenv("PYTHONPATH", [appPath].compactMap { $0 }.joined(separator: ":"), 1)
```

Comment:
If there's only one item on the path, then we don't need to do the `compactMap` transform - we can just use the string verbatim.